[Python] querying articles batch 5 [Replace C# samples] (#1819)
ravendb/docs: Documentation/5.4/Raven.Documentation.Pages/client-api/session/querying/text-search/highlight-query-results.python.markdown

{PANEL/}

{PANEL: Highlight - projected results}

* Highlighting can also be used when [projecting query results](../../../../client-api/session/querying/how-to-project-query-results).

{CODE-TABS}
{CODE-TAB:python:Query highlight_6@ClientApi\Session\Querying\TextSearch\HighlightQueryResults.py /}
{CODE-TAB-BLOCK:sql:RQL}
from "Employees" as x
where search(x.Notes, "manager german")
select { Name : "{0} {1}".format(x.FirstName, x.LastName), Title : x.Title }
include highlight(Notes, 35, 2)
{CODE-TAB-BLOCK/}
{CODE-TABS/}

{CODE:python fragments_3@ClientApi\Session\Querying\TextSearch\HighlightQueryResults.py /}

{PANEL/}

{PANEL: Syntax}

{CODE:python syntax_1@ClientApi\Session\Querying\TextSearch\HighlightQueryResults.py /}

| Parameter          | Type                          | Description                         |
|--------------------|-------------------------------|-------------------------------------|
| **field_name** | `str` | Name of the field that contains the searched terms to highlight |
| **fragment_length** | `int` | Maximum length of a text fragment<br>Must be `>= 18` |
| **fragment_count** | `int` | Maximum number of text fragments that will be returned |

> | **highlightings_callback** | `Callable[[Highlightings], None]` | An 'out' param that will contain the highlighted text fragments for each returned result |

Reviewer: ```suggestion
| **highlightings_callback** | `Callable[[Highlightings], None]` | A callback function (action) ... // like the rest |
```

Author: --> `A callback function to retrieve the highlighted text fragments for each returned result`
